pandas/core/series.py

```diff
@@ -2771,27 +2772,33 @@ def isin(self, values):
 
         See Also
         --------
-        pandas.DataFrame.isin
+        pandas.DataFrame.isin : equivalent method on DataFrame
 
         Examples
         --------
 
-        >>> s = pd.Series(list('abc'))
-        >>> s.isin(['a', 'c', 'e'])
+        >>> s = pd.Series(['lama', 'cow', 'lama', 'beetle', 'lama',
+        ...               'hippo'], name='animal')
```

Comment:
PEP8: add one more space before `'hippo'`

Reply:
updated in 515aecc